packages/core/src/components/slider/multiSlider.tsx

```diff
@@ -99,7 +99,7 @@ export interface ISliderBaseProps extends IProps, IIntentProps {
      *
```

Comment:
we need some more docs here about the new feature...

```suggestion
     *
     * The callback is provided a numeric value and optional rendering options, which include:
     * - isHandleTooltip: whether this label is being rendered within a handle tooltip
     *
```